x-pack/test/functional/apps/infra/config.ts

```diff
@@ -5,20 +5,83 @@
  * 2.0.
  */
 
-import { FtrConfigProviderContext } from '@kbn/test';
-
-export default async function ({ readConfigFile }: FtrConfigProviderContext) {
-  const functionalConfig = await readConfigFile(require.resolve('../../config.base.js'));
+import {
+  ApmSynthtraceEsClient,
+  ApmSynthtraceKibanaClient,
+  InfraSynthtraceEsClient,
+  createLogger,
+  LogLevel,
+} from '@kbn/apm-synthtrace';
+import url from 'url';
+import { FtrConfigProviderContext, kbnTestConfig } from '@kbn/test';
+import { FtrProviderContext as InheritedFtrProviderContext } from '../../ftr_provider_context';
+import { InheritedServices } from './types';
 
+interface MetricsUIConfig {
+  services: InheritedServices & {
+    apmSynthtraceEsClient: (context: InheritedFtrProviderContext) => Promise<ApmSynthtraceEsClient>;
+    infraSynthtraceEsClient: (
+      context: InheritedFtrProviderContext
+    ) => Promise<InfraSynthtraceEsClient>;
+  };
+  pageObjects: any;
+}
+export default async function createTestConfig({
+  readConfigFile,
+}: FtrConfigProviderContext): Promise<MetricsUIConfig> {
+  const baseFunctionalTestsConfig = await readConfigFile(require.resolve('../../config.base.js'));
+  const services = baseFunctionalTestsConfig.get('services');
+  const pageObjects = baseFunctionalTestsConfig.get('pageObjects');
   return {
-    ...functionalConfig.getAll(),
+    ...baseFunctionalTestsConfig.getAll(),
     testFiles: [require.resolve('.')],
+    services: {
+      ...services,
+      infraSynthtraceEsClient: (context: InheritedFtrProviderContext) => {
```

Comment:
not using this client but will be and thought i would make it available now for others